src/repli1d/nn.py

```diff
@@ -95,25 +143,31 @@ def load_signal(name,
     for col in df.columns:
```

Comment:
I think it is better if you put your code: 
```python
  if transform_norm == normal_seq:
      df = pd.DataFrame(transform_norm(df))
      break
```
 before this line as you don't really do any iteration on the column

Reply:
Exactly, thanks.